docs/beta/grain.md

```diff
@@ -6,17 +6,34 @@ image: img/uploads/grain-token-icon.png
 ## What is Grain?
 
 Grain is a community-specific digital currency that is issued on the basis of
-[Cred scores](https://sourcecred.io/docs/beta/cred). Grain represents how much a
-participant has supported that community, either through contributions or
-financial support.
+[Cred scores](https://sourcecred.io/docs/beta/cred).  If cred answers the question
+"*who provided value*?", Grain answers ***"how should we reward people given the value
+they provided?"***
+
+### What is the value of Grain?
+Every community will use Grain differently.  In our community, Grain is currently
+redeemable for Dai, a cryptocurrency that is worth 1 US Dollar.  Communities without
+funds might treat this as a share of future income.  Another might treat it as weight
+behind a community member's vote.
+
+Note that Grain does not _have_ to be tied to any actual monetary reward. A project
+using SourceCred can name it “Points” and use it solely as a scoring system in addition
+to Cred values.
+
+In addition, communities do not have to distribute Grain at all.
 
 Projects can choose to issue their Grain as a cryptocurrency or token, such as
 by issuing Grain as an ERC-20 token on Ethereum. This allows Grain to be a
 conduit with which a project can pay or financially reward participants.
 
-Note that Grain does not _have_ to be tied to any actual monetary concept, such
-as cryptocurrency. A project using SourceCred can name it “Points” and use it
-solely as a scoring system in addition to Cred values.
+### What can I do with Grain?
+Ultimately, Grain is foundations infrastructure for building economics and
+governance mechanisms.  The space of possibility is endless, but here's a few
+ideas:
+- Use Grain to run community non-binding community referendums to get the pulse
+of your community.
+- Build apps that use Grain scores as certificates.  For instance, anyone over
+some Grain score gets edit permissions on Google Docs.
```

Comment:
I'm not 100% sure this is the kind of use of Grain we want to advocate; it feels a more fitting for Cred to be used this way, because Grain is transferable and so in this example, a wealthy Grain owner could just give Grain to select people and take over control of something like this.

Reply:
Yeah, I kind of feel this applies to both examples given here. Cred would be more appropriate in most cases. The main use cases seem to be paying contributors and maybe to use in cryptoeconomic mechanisms like boosting, which enable some form of governance. There are also plenty of other use cases being explored in the crypto space, such as token-gated communities (you have to hold X amount of this token to access this Telegram/Discord), tipping?, etc.

Reply:
Good points.  I replaced the first entirely, and modified the second to use grain staking for permissions, which is unique to grain in that it can be slashed unlike cred (yet).